setup.cfg

```diff
@@ -1,60 +1,41 @@
 [metadata]
-name = packitos
-url = https://github.com/packit-service/packit
-description = A set of tools to integrate upstream open source projects into Fedora operating system.
+name = packit-service
+url = https://github.com/packit-service/packit-service
+description = A service to integrate upstream open source projects into Fedora operating system.
 long_description = file: README.md
 long_description_content_type = text/markdown
 author = Red Hat
 author_email = user-cont-team@redhat.com
 license = MIT
 license_file = LICENSE
 classifiers =
-    Development Status :: 4 - Beta
-    Environment :: Console
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Operating System :: POSIX :: Linux
     Programming Language :: Python
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Topic :: Software Development
-    Topic :: Utilities
 keywords =
     git
     packaging
     fedora
     rpm
     dist-git
+    packit
 
 
 [options]
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.7
 include_package_data = True
 
 setup_requires =
     setuptools_scm
     setuptools_scm_git_archive
 
 install_requires =
-    GitPython
-    PyGithub
     fedmsg
-    libpagure
-    click
-    PyYAML
-    jsonschema
-    ogr
-    rebasehelper
-    munch
     Flask
-    tabulate
-    packaging
-    requests
-    copr
-    python-gnupg
-    # We can't install bodhi-client from PyPI: https://github.com/fedora-infra/bodhi/issues/3058
-    # bodhi-client
```

Comment:
Eviscerate the packages also from [files/install-rpm-packages.yaml](https://github.com/packit-service/packit-service/blob/master/files/install-rpm-packages.yaml)

Reply:
WIP :P